### ISSUE TITLE
Problem Extracting Data With Row Separator

### ISSUE THREAD

**Issue by petercallioni** [tabulapdf/tabula#1052]
I am trying to extract data from a PDF [print-post-sort-plan-september-2019.pdf](https://github.com/tabulapdf/tabula/files/3655468/print-post-sort-plan-september-2019.pdf) which has rows which separate the data as well as columns. 

Using the Lattice method to extra data extracts all of the data perfectly, _except_ for the rows with the red state separator. The state separator is split, where the column line for the rows above and below it are (picture example: 
![image](https://user-images.githubusercontent.com/19599642/65651606-47ea7a80-e052-11e9-93bf-5e7b08b1558c.png))

Using the stream method, only the second page is properly extracted. Example: 
![image](https://user-images.githubusercontent.com/19599642/65651719-a9aae480-e052-11e9-89be-f5567e8a5619.png)

I had the table selection a little larger than the actual size of the table: 
![image](https://user-images.githubusercontent.com/19599642/65651874-1b832e00-e053-11e9-8e1d-769241ca769d.png).

PS. The PDF is publicly available and was gotten from the Australia Post website: https://auspost.com.au/business/marketing-and-communications/business-letter-services/sorting-your-mail

**Comment by jeremybmerrill** [comment]
You might try making separate selections for each state; the row separators ("New south Wales", etc.) are what is confusing Stream mode.

**Comment by petercallioni** [comment]
Yes, that works when using a separate selection for each section, with both Stream and Lattice modes. It should of occurred to me to try this. Thanks for the help. You can close this if you want as the issue has been worked around, but having it differentiate row separators would be a nice feature.
